### PR TITLE
Add Tests for Execute Command Functions

### PR DESCRIPTION
**Pull request by nadeeshaan**

### CODE REVIEW

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java [hunk 1]
```diff
@@ -71,43 +71,53 @@
...
+    private CommandExecutor() {
+    }
+
     /**
      * Command Execution router.
      * @param params            Parameters for the command
      * @param context           Workspace service context
      */
-    public static void executeCommand(ExecuteCommandParams params, LSServiceOperationContext context) {
+    public static Object executeCommand(ExecuteCommandParams params, LSServiceOperationContext context) {
```
Comment (Kishanthan) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java]: We are returning the generic object here? Any reason for that, than specific type?

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java]: @Kishanthan yes. Execute Command request is the language sever side's caller of this particular method. Command execution can for various commands will not return a specific type. That can be vary depending on the command type. Due to this reason WorkspaceService interface's executeCommand method also returns a generic object. 
https://github.com/ballerina-platform/ballerina-lang/blob/master/language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaWorkspaceService.java#L105

Comment (Kishanthan) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java]: Ok, but returning object is not good practice unless no other way. That is my concern. Btw, are using the return value some where?

File: language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/command/CommandExecutionTest.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Command Execution Test Cases.
+ */
```
Comment (Kishanthan) [inline language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/command/CommandExecutionTest.java]: Lets add version

File: language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation1.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                "newText":"\ndocumentation {\n\t\n}\nfunction testDocumentation() {"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "jsonrpc":"2.0"
+  }
+}
```
Comment (Kishanthan) [inline language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation1.json]: eof line missing

File: language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation2.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                "newText":"import ballerina/io;\n\ndocumentation {\n\t\n}\nfunction testDocumentation() {"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "jsonrpc":"2.0"
+  }
+}
```
Comment (Kishanthan) [inline language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation2.json]: eof line missing, please fix everywhere applicable

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java (hunk 1 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java]: Yes, Usually this is used depending on what type of command, by the Language Clients such as VSCode and etc.

File: language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/command/CommandExecutionTest.java (hunk 2 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/command/CommandExecutionTest.java]: Fixed with 28bb5311467e02cc5504b51009bd9716cca8841e

File: language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation1.json (hunk 3 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation1.json]: Fixed with 28bb5311467e02cc5504b51009bd9716cca8841e

File: language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation2.json (hunk 4 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/resources/command/addSingleFunctionDocumentation2.json]: fixed with 28bb5311467e02cc5504b51009bd9716cca8841e

**Comment by Kishanthan** [comment]
LGTM

Reg the text : @since v0.982.0
-we don't use "v"
-also there should be new line after calls description before the version tag

**Comment by nadeeshaan** [comment]
Fixed the version doc comment with #10162

**Pull request merged**